<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="sortB">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="drawGraph">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Draw graphic</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>4</number>
        </property>
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>90</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>33</number>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
         <property name="flags">
          <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="4">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="5">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>89</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
